generator/docs/GenerationTypes.md

```diff
@@ -3,10 +3,11 @@
 The generator supports the following data generation types
 
 * Random (_default_)
+* Full_sequential
```

Comment:
We should leave this feature as hidden for now - by design - as there are cases where it doesn't produce the expected set of results.